2023-01-03 17:49:29,077 (33963/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-01-03 17:49:29,077 (33963/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-01-03 17:49:37,632 (33963/MainThread) pgcli.pgexecute ERROR - sql: 'SELECT * FROM notifcations', error: UndefinedTable('relation "notifcations" does not exist\nLINE 1: SELECT * FROM notifcations\n                      ^')
2023-01-03 17:49:37,635 (33963/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 373, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 423, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 69, in execute\n    super().execute(*args, **kwargs)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/psycopg/cursor.py", line 725, in execute\n    raise ex.with_traceback(None)\npsycopg.errors.UndefinedTable: relation "notifcations" does not exist\nLINE 1: SELECT * FROM notifcations\n                      ^\n'
2023-01-04 15:54:42,218 (31570/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-01-04 15:54:42,220 (31570/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-01-04 15:55:04,026 (31570/MainThread) pgcli.pgexecute ERROR - sql: '\\ddt', error: SyntaxError('syntax error at or near "\\"\nLINE 1: \\ddt\n        ^')
2023-01-04 15:55:04,030 (31570/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 373, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 423, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 69, in execute\n    super().execute(*args, **kwargs)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/psycopg/cursor.py", line 725, in execute\n    raise ex.with_traceback(None)\npsycopg.errors.SyntaxError: syntax error at or near "\\"\nLINE 1: \\ddt\n        ^\n'
2023-01-04 15:55:04,865 (31570/MainThread) pgcli.pgexecute ERROR - sql: '\\]dt', error: SyntaxError('syntax error at or near "\\"\nLINE 1: \\]dt\n        ^')
2023-01-04 15:55:04,867 (31570/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 373, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 423, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 69, in execute\n    super().execute(*args, **kwargs)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/psycopg/cursor.py", line 725, in execute\n    raise ex.with_traceback(None)\npsycopg.errors.SyntaxError: syntax error at or near "\\"\nLINE 1: \\]dt\n        ^\n'
2023-01-04 15:55:29,703 (32240/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-01-04 15:55:29,703 (32240/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-01-04 15:56:15,637 (32240/MainThread) pgcli.pgexecute ERROR - sql: 'SELECT\n* FROM users', error: UndefinedTable('relation "users" does not exist\nLINE 2: * FROM users\n               ^')
2023-01-04 15:56:15,644 (32240/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 373, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 423, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 69, in execute\n    super().execute(*args, **kwargs)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/psycopg/cursor.py", line 725, in execute\n    raise ex.with_traceback(None)\npsycopg.errors.UndefinedTable: relation "users" does not exist\nLINE 2: * FROM users\n               ^\n'
2023-01-04 15:58:02,573 (33001/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-01-04 15:58:02,574 (33001/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-01-04 15:58:16,672 (33001/MainThread) pgcli.pgexecute ERROR - sql: 'SELECT * FROM matching_v3', error: UndefinedTable('relation "matching_v3" does not exist\nLINE 1: SELECT * FROM matching_v3\n                      ^')
2023-01-04 15:58:16,676 (33001/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 373, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 423, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 69, in execute\n    super().execute(*args, **kwargs)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/psycopg/cursor.py", line 725, in execute\n    raise ex.with_traceback(None)\npsycopg.errors.UndefinedTable: relation "matching_v3" does not exist\nLINE 1: SELECT * FROM matching_v3\n                      ^\n'
2023-01-04 15:58:33,970 (33001/MainThread) pgcli.pgexecute ERROR - sql: 'SELECT * FROM public', error: UndefinedTable('relation "public" does not exist\nLINE 1: SELECT * FROM public\n                      ^')
2023-01-04 15:58:33,972 (33001/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 373, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 423, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 69, in execute\n    super().execute(*args, **kwargs)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/psycopg/cursor.py", line 725, in execute\n    raise ex.with_traceback(None)\npsycopg.errors.UndefinedTable: relation "public" does not exist\nLINE 1: SELECT * FROM public\n                      ^\n'
2023-01-04 15:58:47,482 (33001/MainThread) pgcli.pgexecute ERROR - sql: 'SELECT * FROM public.xbeam_match_time()', error: UndefinedFunction('function public.xbeam_match_time() does not exist\nLINE 1: SELECT * FROM public.xbeam_match_time()\n                      ^\nHINT:  No function matches the given name and argument types. You might need to add explicit type casts.')
2023-01-04 15:58:47,483 (33001/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 373, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 423, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 69, in execute\n    super().execute(*args, **kwargs)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/psycopg/cursor.py", line 725, in execute\n    raise ex.with_traceback(None)\npsycopg.errors.UndefinedFunction: function public.xbeam_match_time() does not exist\nLINE 1: SELECT * FROM public.xbeam_match_time()\n                      ^\nHINT:  No function matches the given name and argument types. You might need to add explicit type casts.\n'
2023-01-04 16:03:29,663 (33001/MainThread) pgcli.pgexecute ERROR - sql: 'SELECT * FROM population_matches_v3', error: UndefinedTable('relation "population_matches_v3" does not exist\nLINE 1: SELECT * FROM population_matches_v3\n                      ^')
2023-01-04 16:03:29,664 (33001/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 373, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 423, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 69, in execute\n    super().execute(*args, **kwargs)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/psycopg/cursor.py", line 725, in execute\n    raise ex.with_traceback(None)\npsycopg.errors.UndefinedTable: relation "population_matches_v3" does not exist\nLINE 1: SELECT * FROM population_matches_v3\n                      ^\n'
2023-01-04 16:03:50,797 (34043/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-01-04 16:03:50,798 (34043/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-01-04 16:06:00,288 (34172/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-01-04 16:06:00,289 (34172/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-01-04 16:06:10,506 (34187/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-01-04 16:06:10,506 (34187/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-01-05 16:56:58,539 (12710/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-01-05 16:56:58,540 (12710/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-01-09 16:03:48,502 (52944/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-01-09 16:03:48,502 (52944/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-01-10 10:55:57,361 (87401/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-01-10 10:55:57,362 (87401/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-01-11 15:54:22,037 (91033/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-01-11 15:54:22,037 (91033/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-01-11 15:54:38,524 (91033/MainThread) pgcli.pgexecute ERROR - sql: 'SELECT * FROM source_fields WHERE', error: SyntaxError('syntax error at end of input\nLINE 1: SELECT * FROM source_fields WHERE\n                                         ^')
2023-01-11 15:54:38,529 (91033/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 373, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 423, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 69, in execute\n    super().execute(*args, **kwargs)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/psycopg/cursor.py", line 725, in execute\n    raise ex.with_traceback(None)\npsycopg.errors.SyntaxError: syntax error at end of input\nLINE 1: SELECT * FROM source_fields WHERE\n                                         ^\n'
2023-01-17 10:16:21,840 (74615/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-01-17 10:16:21,841 (74615/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-01-17 10:17:40,329 (75184/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-01-17 10:17:40,330 (75184/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-01-17 10:23:06,399 (75184/MainThread) pgcli.main ERROR - sql: 'SELECT * FROM sources WHERE feed_id = 5614;', error: StopIteration()
2023-01-17 10:23:06,405 (75184/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/main.py", line 705, in execute_command\n    output, query = self._evaluate_command(text)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/main.py", line 1002, in _evaluate_command\n    formatted = format_output(\n                ^^^^^^^^^^^^^^\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/main.py", line 1575, in format_output\n    first_line = next(formatted)\n                 ^^^^^^^^^^^^^^^\nStopIteration\n'
